.github/blunderbuss.yml

```diff
@@ -5,17 +5,114 @@ assign_issues_by:
   - engelke
 - labels:
   - 'api: automl'
+  - 'api: dialogflow'
+  - 'api: language'
+  - 'api: speech'
+  - 'api: texttospeech'
+  - 'api: translate'
+  - 'api: video'
+  - 'api: vision'
   to:
   - sirtorry
   - telpirion
 - labels:
   - 'api: bigquery'
+  - 'api: datacatalog'
   to:
   - shollyman
 - labels:
   - 'api: bigtable'
   to:
   - billyjacobson
+- labels:
+  - 'api: billingbudgets'
+  to:
+  - GoogleCloudPlatform/billing-samples-maintainers
+- labels:
+  - 'api: cloudsql'
+  to:
+  - GoogleCloudPlatform/cloud-sql-dpes
+- labels:
+  - 'api: containeranalysis'
+  to:
+  - donmccasland
+- labels:
+  - 'api: datalabeling'
+  - 'api: iot'
+  to:
+  - sgreenberg
+- labels:
+  - 'api: dataproc'
+  to:
+  - bradmiro
+- labels:
+  - 'api: datastore'
+  - 'api: firestore'
+  to:
+  - GoogleCloudPlatform/firestore-dpe
+- labels:
+  - 'api: dns'
+  to:
+  - michaelawyu
+- labels:
+  - 'api: functions'
+  to:
+  - ace-n
+  - grant
+- labels:
+  - 'api: healthcare'
+  to:
+  - noerog
+- labels:
+  - 'api: iam'
+  to:
+  - melaniedejong
+- labels:
+  - 'api: logging'
+  to:
+  - simonz130
```

Comment:
Can we autoassign one person (fine if it's me) and also add a team as a subscriber to the bug?

Reply:
According to https://github.com/googleapis/repo-automation-bots/tree/master/packages/blunderbuss

Probably not possible.

Reply:
If you add the team in the CODEOWNER, they'll be subscribed when the bug opens.